cql3/constants.hh

```diff
@@ -73,7 +73,7 @@ public:
         value(cql3::raw_value bytes_) : _bytes(std::move(bytes_)) {}
         virtual cql3::raw_value get(const query_options& options) override { return _bytes; }
         virtual cql3::raw_value_view bind_and_get(const query_options& options) override { return _bytes.to_view(); }
-        virtual sstring to_string() const override { return to_hex(*_bytes); }
+        virtual sstring to_string() const override { return to_hex(to_bytes(_bytes.to_view())); }
```

Comment:
Isn't this a regression? We'll now linearize unconditionally. Although it's already bad, since the returned string is linear and 2X larger.

Reply:
I assumed that to_string() exists for debugging purposes and glossed over it. But fixing it is trivial, we just need to_hex_fragmented.

Reply:
Well, you're right. It's not exactly debugging, it's diagnostics, used in error messages sent back to the user. It's not worth optimizing.

Later, we should remove bytes and to_bytes and clear it, so that even diagnostics don't hurt the workload. But right now it's lost in the noise.